src/renderer/components/dock/terminal.store.ts

```diff
@@ -120,7 +120,7 @@ export class TerminalStore extends Singleton {
       const rcIsFinished = when(() => this.connections.get(tab.id).shellRunCommandsFinished);
       const notifyVeryLong = setTimeout(() => {
         rcIsFinished.cancel();
-        Notifications.info("Terminal shell is taking a long time to complete startup. Please check your .rc file. Bypassing shell completion check.", {
+        Notifications.info("Terminal shell is taking a long time to complete startup. Bypassing shell completion check.", {
```

Comment:
Or:
```suggestion
        Notifications.info("Terminal shell is taking a long time to complete startup. Please check your .rc file, if applicable. Bypassing shell completion check.", {
```

Reply:
Fair enough

Reply:
Maybe something like:
```suggestion
        Notifications.info("If terminal shell is not ready please check your shell init files, if applicable. You may see this message if you have set a custom shell prompt", {
```

Reply:
Sounds good, nice and system agnostic.